jobqueue/src/main/java/com/path/android/jobqueue/network/NetworkUtilImpl.java

```diff
@@ -36,7 +38,16 @@ public boolean isConnected(Context context) {
             return false;
         }
 
+        int permState = context.checkCallingOrSelfPermission(Manifest.permission.ACCESS_NETWORK_STATE);
+        if (permState != PackageManager.PERMISSION_GRANTED) {
```

Comment:
Hi @yigit!

I re-created the PR on top of master. Here it is: https://github.com/yigit/android-priority-jobqueue/pull/138.